docs/serving/getting-started-knative-app.md

```diff
@@ -36,24 +35,41 @@ docker tag local-image dev.local/local-image
 
 [Learn more about image tag resolution.](./tag-resolution.md)
 
-The Hello World sample app reads in an `env` variable, `TARGET`, from the
-configuration `.yaml` file, then prints "Hello World: \${TARGET}!". If `TARGET`
-isn't defined, it will print "NOT SPECIFIED".
+The Hello World sample app reads in an `env` variable, `TARGET`, then prints "Hello World: \${TARGET}!". If `TARGET` isn't defined, it will print "NOT SPECIFIED".
 
-## Configuring your deployment
+## Creating your Deployment with the Knative CLI
 
-To deploy an app using Knative, you need a configuration `.yaml` file that
-defines a Service. For more information about the Service object, see the
+The easiest way to create Knative service deployment is by using the Knative CLI [kn](https://github.com/knative/client).
+You can install the `kn` binary as described in [Installing the Knative CLI](../install/install-kn.md)
+
+`kn` allows you to create a service interactively on the command line.
+Eventually, it will create a corresponding resource description internally as when using a YAML file directly.
+
+To create our service directly at the cluster, use:
+
+```shell
+# Create a Knative service with the Knatice CLI kn
+kn service create helloworld-go --image gcr.io/knative-samples/helloworld-go --env TARGET="Go Sample v1"
```

Comment:
The other section mentions that you can reference a different image in the YAML. Do you think we need similar text here?

Reply:
Good point, I will copy that over to the `kn` section.

Reply:
BTW, the section for the YAML files is

```
If you want to deploy the sample app, leave the config file as-is. If you're
deploying an image of your app, update the name of the app and the URL of
the image accordingly.
```

If ok, I will fix that, too, as the image value is not really an URL but a reference to a container image in 'Docker syntax'

Reply:
Added now the following to the YAML section:

-----

If you want to deploy the sample app, leave the config file as-is. If you're
deploying an image of your app, update the name of the Service (`.metadata.name`) and the reference to the container image (`.spec.containers[].image`) accordingly.

-----

+ the image comment in the yaml